drivers/display/Kconfig.sdl

```diff
@@ -46,4 +46,15 @@ config SDL_DISPLAY_ZOOM_PCT
 	help
 	  SDL window zoom percentage to adjust readability on small screens
 
+config SDL_DISPLAY_USE_SOFTWARE_RENDERER
+	bool "Use software renderer"
+	help
+	Use software renderer instead of hardware accelerated renderer
+
+config SDL_DISPLAY_HIDE_WINDOW
+	bool "Use software renderer"
```

Comment:
Copy paste error

Reply:
Thank you for pointing it out.
I rewrote it as a whole about added configs.